CIP-????/README.md

```diff
@@ -0,0 +1,112 @@
+---
+CIP: ?
+Title: Dynamic Asset Resale Fees
+Status: Proposed
+Category: Tokens
+Authors:
+  - Brock <brockcruess@gmail.com>
+Implementors: N/A
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/116
+Created: 2024-01-09
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposed standard will serve as a more functional alternative to [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027), allowing resale fee (previously incorrectly referred to as "royalty") information to be updated at any time - natively for unlocked policies, or referenced by [CIP-0088](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0088) for a locked policy. This proposed standard will also allow for multiple fee rates and output destinations to be built into resale transactions. This proposal is essentially a rewrite of [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027) to maintain similar structure and easy comparison, so some of the wording/formatting is courtesy of the [CIP-0027 Authors](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027) but is likely modified to highlight the differences between [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027) and this proposed CIP.
+
+## Motivation: Why is this CIP necessary?
+
+[CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027) provides great groundwork for resale fee functionality, but lacks support for updating fee rates/destination addresses, and also lacks support for multiple fee outputs. Projects may choose to maintain multiple wallets and split resale fees between them. DAOs may want to to govern their own asset collection's resale fee rates and their fee destinations. A use-case example would be a digital collectible project that wants to charge resale fees of 4% to a DAO wallet and 1% to a core team wallet. Another example would be an artist who wants to, just for the next 30 days, send all resale fees of their minted assets to a charity wallet, and decrease their policy's fee rate during that time as well. Because this new standard changes two features of [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027), it is being proposed as a new CIP rather than as an update to [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027).
+
+## Specification
+
+- A new tag of 778 is proposed for this implementation. The 778 tag compliments [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027)'s 777 tag.
+- Any unlocked policy can write initial or updated resale fees tags to a new token.
+- The resale fees tags are to be written to an unnamed token, using the policy to be used for the intended Cardano Assets.
```

Comment:
requiring the "same signer as the policy", for a native-scritpt policy, even if that policy is locked seems to be the expectation.  Is that expectation present elsewhere and could be referenced by abstract?  Or should it be explicated here?

Reply:
yes, that's the expectation to prove "ownership". it was my understanding that CIP-88 takes this approach, but @Crypto2099 could clarify on that.